### PR TITLE
Fix/ADF-532/Fix DAC annotation which blocks class property deletion

### PR DESCRIPTION
**Pull request by shpran**
**Relates to:** _[ADF-532](https://oat-sa.atlassian.net/browse/ADF-532)_

**Changes:**
* Fixed invalid annotation;
* Added log to check actions access;
* Fixed test.

**Require:** _installed `taoDacSimple`._

**How to test:**
1. Open Items tab;
2. Select any class;
3. Open `Schema Management`;
4. Add new property;
5. Try to delete it;
6. Click save or reload the page and reopen `Schema Management`.

**Bug:** _property was not deleted._
**Fix:** _property successfully deleted._

### CODE REVIEW

File: models/classes/accessControl/ActionAccessControl.php [hunk 1]
```diff
@@ -147,6 +147,18 @@ private function hasAccess(array $allowedPermissions, string $controller, string
             }
         }
 
+        if (!$roleIsListed) {
+            $this->logWarning(
+                sprintf(
+                    'User roles "%s", does not have permissions "%s" form controller "%s" and action "%s',
```
Comment (gabrielfs7) [inline models/classes/accessControl/ActionAccessControl.php:153]: Small typo, please blame who sent this code :p 

```suggestion
                    'User roles "%s" do not have permissions "%s" form controller "%s" and action "%s',
```

**Review by gabrielfs7 — APPROVED** [review]
LGTM. Lets please test with UDIR before merge

**Review by boajer — APPROVED** [review]

**Review by chinnu-focaloid — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [X] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-shpran.playground.kitchen.it.taocloud.org:41481/

**Pull request merged**